src/main/java/uk/gov/hmcts/reform/unspec/handler/callback/user/AddOrAmendClaimDocumentsCallbackHandler.java

```diff
@@ -28,7 +28,7 @@
     protected Map<String, Callback> callbacks() {
         return Map.of(
             callbackKey(ABOUT_TO_START), this::emptyCallbackResponse,
-            callbackKey(MID, "particulars-of-claim"), this::validateParticularsOfClaim,
+            callbackKey(MID, "particulars-of-claim"), this::validateParticularsOfClaimAddOrAmendDocuments,
```

Comment:
is this the right place? I thought this story was for claimIssue event?

Reply:
this story involves changing the validation in addOrAmendClaimDocuments event